code/API_definitions/qos-profiles.yaml

```diff
@@ -0,0 +1,475 @@
+      apiRoot:
+        default: http://localhost:9091
+        description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
+tags:
+  - name: QoS Profiles
+    description: Manage QoS Profiles
+paths:
+  /qos-profiles:
+    get:
+      tags:
+        - QoS Profiles
+      summary: "Get All QoS Profiles"
+      description: |
+        Returns all QoS Profiles that match the given criteria.
+        If no criteria is given, all QoS Profiles are returned.
+      operationId: getQosProfiles
+      parameters:
+        - name: name
+          in: query
+          description: QoS Profile name
+          schema:
+            type: string
+          required: false
+        - name: status
+          in: query
+          description: Qos Profile status
+          schema:
+            $ref: '#/components/schemas/QosProfileStatusEnum'
+          required: false
+        - $ref: "#/components/parameters/x-correlator"
+      responses:
+        "200":
+          description: Contains information about QoS Profiles
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/QosProfile"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/QosProfilesNotFound404"
+        "500":
+          $ref: "#/components/responses/QoSProfile500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /qos-profiles/{name}:
+    get:
+      tags:
+        - QoS Profiles
+      summary: "Get QoS Profile for a given name"
+      operationId: getQosProfile
+      description: |
+        Returns a QoS Profile that matches the given name.
+      parameters:
+        - name: name
+          in: path
+          description: Qos Profile name
+          required: true
+          schema:
+            $ref: "#/components/schemas/QosProfileName"
+        - $ref: "#/components/parameters/x-correlator"
+      responses:
+        "200":
+          description: Contains information about QoS Profiles
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QosProfile"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/QosProfileNotFound404"
+        "500":
+          $ref: "#/components/responses/QoSProfile500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+components:
```

Comment:
```suggestion
components:
  securitySchemes:
    oAuth2ClientCredentials:
      description: |
        The QoS Profiles API makes use of the OAUTH 2.0 client credentials grant which is applicable for server to server use cases involving trusted partners or clients without any protected user data involved. In this method the API invoker client is registered as a confidential client with an authorization grant type of client_credentials
      type: oauth2
      flows:
        clientCredentials:
          tokenUrl: https://api.example.com/oauth/token
          scopes: {}

```

Reply:
Agree. As the intention of the PR is just to split the current version, I think it's fine to keep by now the same definition. We will have to align all security related parts to the Guidelines before the meta-release.